package.json

```diff
@@ -27,12 +27,13 @@
   },
   "scripts": {
     "ng": "ng",
+    "clean:workspace": "del ./**/dist && yarn clean:modules",
     "clean:modules": "del ./**/node_modules",
     "clean": "del ./dist && yarn run schematics:clean",
```

Comment:
To simplify a bit I think `clean:workspace` and `clean:modules` could be combined into just `clean:workspace`. If you run `clean:modules` you should rebuild anyways since new dependencies would be installed.